client-src/index.js

```diff
@@ -10,12 +10,20 @@ import sendMessage from "./utils/sendMessage.js";
 import reloadApp from "./utils/reloadApp.js";
 import createSocketURL from "./utils/createSocketURL.js";
 
+/**
+ * @typedef {Object} OverlayOptions
+ * @property {boolean | (error: Error) => boolean} [warnings]
```

Comment:
@alexander-akait When adding tests I realize that the error here can be `string` in webpack4. Should I update the type to reflect that?

Reply:
hm, is it only type problem? Because if yes, no need, I want to do this release and focus on the next major release, drop old node.js version, remove deprecation and unnecessary options and etc and drop webpack v4